app/views/users/show.html.haml

```diff
@@ -113,6 +113,10 @@
               - @admin_ontologies.each do |ont|
                 .account-page-submitted-ontology{data: {controller: 'tooltip'}, title: ont.name}
                   %a{href: "/ontologies/#{ont.acronym}"}= ont.acronym
+                - unless ont.views.nil? || ont.views.empty?
+                  - ont.views.each do |view|
```

Comment:
This solution is not good.

The goal is not to show the views of the submitted ontologies.

But to show all the submitted ontologies and views of the users (currently only the ontologies are listed)

Reply:
@SirineMhedhbi  remove this code and open a new PR